pyccel/decorators.py

```diff
@@ -139,7 +173,19 @@ class KernelAccessor:
         def __init__(self, f):
             self._f = f
         def __getitem__(self, args):
-            return self._f
+            num_blocks, num_threads = args
```

Comment:
Here too, num_blocks and num_threads are not necessarily one dimentional and can be up to three dimensions

Reply:
For the moment, you can't use tuples for num_blocks and num_threads. If you pass these values as tuples to the kernel call, you will receive a 'not implemented' error. Once this issue is resolved, I will implement the logic for the dim